components/Feedback.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { Flex, Box, Button, Text } from '@100mslive/react-ui';
+import useClickOutside from '@/lib/useClickOutside';
+import { currentUser } from '../lib/currentUser';
+
+const emojis = [{ score: 1 }, { score: 2 }, { score: 3 }, { score: 4 }];
+
+const Feedback = () => {
+    const [showTextBox, setShowTextBox] = React.useState(false);
+    const [clickedEmoji, setClickedEmoji] = React.useState(0);
+    const [submitSuccessful, setSubmitSuccessful] = React.useState(false);
+    const [message, setMessage] = React.useState('');
+    const feedBackRef = React.useRef<HTMLDivElement | null>();
+    const inputRef = React.useRef<HTMLTextAreaElement | undefined>();
+
+    // @ts-ignore
+    useClickOutside(feedBackRef, () => setShowTextBox(false));
+
+    React.useEffect(() => {
+        if (inputRef.current) inputRef.current.focus();
+    }, [clickedEmoji]);
+
+    return (
+        // @ts-ignore
+        <Box ref={feedBackRef} css={{ maxWidth: '200px' }}>
+            <Text
+                variant="tiny"
+                css={{ fontWeight: 'bold', color: '$textHighEmp', textTransform: 'uppercase' }}>

```

Comment:
these styles are slightly different from the styles on `on this page`. ones i could find:

* font weight should be 500 (bold gives 700)
* letter spacing needs to be higher